app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -139,6 +139,11 @@
       = link_to(admin_conference_roles_path(@conference.short_title)) do
         %span.fa.fa-group
         Roles
+  - if can? :index, PhysicalTicket.new(conference: @conference)
```

Comment:
I would say this is not the best place for that. We have a donation section, which has tickets on it. It could be a button inside the ticket (which is inside donations) section. I think we don't need to add this to the sidebar, which is getting quite long. :thinking: